numba/numba_entry.py

```diff
@@ -123,6 +124,32 @@ def get_sys_info():
                 print(
                     "Error: Probing CUDA failed (device and driver present, runtime problem?)\n")
 
+        print("")
+        print("__SVML Information__")
+        # replicate some SVML detection logic from numba.__init__ here.
+        # if SVML load fails in numba.__init__ the splitting of the logic
+        # here will help diagnosis of the underlying issue
+        have_svml_library = True
+        try:
+            if sys.platform.startswith('linux'):
+                llvmbind.load_library_permanently("libsvml.so")
+            elif sys.platform.startswith('darwin'):
+                llvmbind.load_library_permanently("libsvml.dylib")
+            elif sys.platform.startswith('win'):
+                llvmbind.load_library_permanently("svml_dispmd")
+            else:
+                have_svml_library = False
+        except:
+            have_svml_library = False
```

Comment:
Isn't it loaded already and the `have_svml_library = numba.config.USING_SVML`

Reply:
What if you set the env var that disables SVML, the `config.USING_SVML` should respond as `False` but the library finding etc ought to still report true?

Reply:
ok, you are right. I think that that with few refactoring, it will be possible to reuse `_try_enable_svml()`